docs/usage/quickstart.rst

```diff
@@ -3,112 +3,112 @@
-    node-03
+         [kube-master]
+         node-01
+         node-02
+         node-03
 
-    [kube-node]
-    node-01
-    node-02
-    node-03
+         [etcd]
+         node-01
+         node-02
+         node-03
 
-    [k8s-cluster:children]
-    kube-node
-    kube-master
+         [kube-node]
+         node-01
+         node-02
+         node-03
 
-Make sure to change IP-addresses, usernames etc. according to your
-infrastructure.
+         [k8s-cluster:children]
+         kube-node
+         kube-master
 
-In a second file, called :file:`kube-node.yml` in a :file:`group_vars`
-subdirectory of our inventory, we declare how to setup storage (in the
-default configuration) on hosts in the *kube-node* group, i.e. hosts on which
-Pods will be scheduled:
+    Make sure to change IP addresses, user names, etc. to conform to your
+    infrastructure.
 
-.. code-block:: yaml
+  - A second file, called :file:`kube-node.yml`, in a :file:`group_vars`
+   subdirectory of the inventory, declares how to set up storage (in the
+   default configuration) on hosts in the *kube-node* group, that is,
+   hosts on which Pods will be scheduled::
 
-    metal_k8s_lvm:
-      vgs:
-        kubevg:
-          drives: ['/dev/vdb']
+         metal_k8s_lvm:
+           vgs:
+             kubevg:
+               drives: ['/dev/vdb']
 
-In the above, we assume every *kube-node* host has a disk available as
+In the above, every *kube-node* host is assumed to have a disk available as
 :file:`/dev/vdb` which can be used to set up Kubernetes *PersistentVolumes*. For
 more information about storage, see :doc:`../architecture/storage`.
 
 Entering the MetalK8s Shell
 ---------------------------
-To easily install a supported version of Ansible and its dependencies, as well
-as some Kubernetes tools (:program:`kubectl` and :program:`helm`), we provide a
-:program:`make` target which installs these in a local environment. To enter this
+To install a supported version of Ansible and its dependencies, along with some
+Kubernetes tools (:program:`kubectl` and :program:`helm`), create a
+:program:`make` target that installs these in a local environment. To enter this
 environment, run :command:`make shell` (this takes a couple of seconds on first
 run)::
 
-    $ make shell
-    Creating virtualenv...
-    Installing Python dependencies...
-    Downloading kubectl...
-    Downloading Helm...
-    Launching metal-k8s shell environment. Run 'exit' to quit.
-    (metal-k8s) $
+      $ make shell
+      Creating virtualenv...
+      Installing Python dependencies...
+      Downloading kubectl...
+      Downloading Helm...
+      Launching metal-k8s shell environment. Run 'exit' to quit.
+      (metal-k8s) $
 
-Now we're all set to deploy a cluster::
+Now, you're all set to deploy a cluster::
 
     (metal-k8s) $ ansible-playbook -i inventory/quickstart-cluster -b metal-k8s.yml
 
 Grab a coffee and wait for deployment to end.
 
-Inspecting the cluster
+Inspecting the Cluster
```

Comment:
`pod`, `deployment` but `Cluster`? Or is this title-specific?

Reply:
I uppercased the headings.